### ISSUE TITLE
More Platform types

### ISSUE THREAD

**Issue by Jontpan** [AxlLind/2D-platformer#96]
We should add at least a couple more platform types, four feels very little

**Comment by Jontpan** [comment]
I'm currently working on a platform which makes the next four be invisible until you jump on them.
Are there any other that we should have? I'm also thinking of adding moving trampoline, and one that disappears after you jump on it. @AxlLind

**Comment by AxlLind** [comment]
Sounds good. Moving platforms could be changed to be a property of all platforms? As in a boolean 'moving' on the standard platformScript.

Would we have to make sure that the platforms that are invisible aren't moving? Otherwise it sound very, very difficult.

**Comment by Jontpan** [comment]
Yeah they would not be moving. It is very difficult, yes, but it could probably be made easier if in testing it turns out too difficult. 

Having a boolean moving would make some things easier, but since the platformspawner just spawns the prefabs it makes more sense to have moving trampoline as a prefab.
Otherwise on awake the platform would have to decide for itself if its moving or not, rather than have those percentages on platformspawner. It would kinda mess things up.

**Comment by AxlLind** [comment]
Yes, you could do it in the awake but I think we want the chance for it to be moving to increase with y-pos so it might make more sense in the platformSpawner. Since the spawn prefab function returns the gameobject you could just access the platform script component and set a public variable moving to true?

**Comment by Jontpan** [comment]
Well, what I mean is, we would have percentages in multiple places. The percentages for a specific prefab to be chosen and then in a separate function the percentage for that platform to be moving? It's not hard to just make a separate prefab for the moving ones.

**Comment by Jontpan** [comment]
I'm also gonna change some logic in platformspawner, to make use of a queue.
